README.md

```
@@ -10,11 +10,12 @@
 -   [Integration Details](#integration-details)
 -   [Workflows](#workflows)
 -   [Data Flow and Storage](#data-flow-and-storage)
+-   [Planned Features & Roadmap](#planned-features--roadmap)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid

(MD051, link-fragments)